metadata.rb

```diff
@@ -10,7 +10,7 @@
 
 depends          'base', '>= 2.6.0'
 depends          'osl-git'
-depends          'java', '< 8.0'
+depends          'java'
```

Comment:
This is the first release which included `openjdk_pkg_install` so we should ensure we use that version.
```suggestion
depends          'java', '>= 8.1.0'
```